erpnext/buying/doctype/purchase_order/purchase_order.json

```diff
@@ -1130,11 +1130,11 @@
    "write": 1
   }
  ],
- "search_fields": "status, transaction_date, supplier,grand_total",
+ "search_fields": "status, transaction_date, supplier, grand_total",
  "show_name_in_global_search": 1,
  "sort_field": "modified",
  "sort_order": "DESC",
  "timeline_field": "supplier",
- "title_field": "supplier",
+ "title_field": "supplier_name",
```

Comment:
from what i see `supplier_name` is only populated if the Supplier Code and Name differ. It is blank if they are the same. So keeping it `supplier` is best

Reply:
Let's change that as well. Can you point me to the code you're talking about?

Reply:
Hey really sorry, it just toggles supplier_name's display, the value still remains. This works 👍

Reply:
I cannot reproduce what you're saying on my instance. `supplier_name` get's set even when it is identical to `supplier`. It is just hidden to not show duplicate information, I guess.